crates/fj-app/Cargo.toml

```diff
@@ -10,6 +10,7 @@ repository = "https://github.com/hannobraun/fornjot"
 license = "0BSD"
 keywords = ["cad", "programmatic", "code-cad"]
 categories = ["encoding", "mathematics", "rendering"]
+build = "build.rs"
```

Comment:
Should be unnecessary, no? At least I'm pretty sure I've never added this, beside working with build scripts before.

Reply:
To be clear, this is just me wondering out loud. No change needs to be made.

Reply:
Yeah, as of a couple years ago `cargo` will automatically run a `build.rs` if it finds one next to `Cargo.toml`. I've always put it in there to be explicit, but I guess it's not really necessary.